### PR TITLE
Feat/hospital screen

### PR DESCRIPTION
**Pull request by yunior05**
Add hospital & laboratory screen

#### Linked issues:

Fixes [#44](https://trello.com/c/X3nsgeY0/44-hospital-map-view) & [#47](https://trello.com/c/qlye058L/47-vista-del-mapa-de-laboratorios)

#### Screenshots:

Screen Shot 2020-05-15 at 1 18 07 PM

Screen Shot 2020-05-15 at 3 43 20 PM

#### How to test

- `git fetch && git checkout feat/hospital-screen`
- Go to options => Hospital Map
- Test design and functionality
- Repeat with Laboratories Map

### CODE REVIEW

**Review by adnanjt — APPROVED** [review]
app/components/ButtonUpDrawer.js

If this is a new file it should go under DR folder.

File: app/components/ButtonUpDrawer.js [hunk 1]
```diff
@@ -0,0 +1,214 @@
+/* eslint-disable react/state-in-constructor */
+/* eslint-disable react/static-property-placement */
+/* eslint-disable react/no-deprecated */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/sort-comp */
```
Comment (iocedano) [inline app/components/ButtonUpDrawer.js]: Why?

Comment (yunior05) [reply app/components/ButtonUpDrawer.js]: sorry, forget to remove that eslint comments

**Review by inteljudajocu — APPROVED** [review]
Comments have been addressed

**Pull request merged**